og.module

```diff
@@ -95,6 +95,27 @@ function og_entity_access(EntityInterface $entity, $operation, AccountInterface
 }
 
 /**
+ * Implements hook_entity_insert().
+ */
+function og_entity_insert(EntityInterface $entity) {
+  if (!\Drupal\og\Og::isGroup($entity->getEntityTypeId(), $entity->bundle())) {
+    return;
+  }
+
+  if (!empty($entity->hasField('uid'))) {
```

Comment:
maybe, although I'm trying to follow the "is there a real use case" approach. 98% of groups are nodes. Rest I know of are terms and users.

Anyway, nice to have but not a priority.